frontend/package.json

```diff
@@ -46,7 +48,6 @@
     "@glimmer/component": "1.1.2",
     "@glimmer/tracking": "1.1.2",
     "@html-next/vertical-collection": "4.0.2",
-    "@sentry/ember": "^7.98.0",
```

Comment:
Why remove sentry? It was the goal to enable it some time.

Reply:
i did some local testing, i split these changes into seperate PRs